bam_to_mate_hist.py

```diff
@@ -149,19 +164,71 @@ def calc_n50_from_header(header, xx=50.0):
     cumsum = 0
     contig_len = 0
     for length in reversed(lens):
-        contig_len = length
+        contig_len = int(length)
         cumsum += length
         if cumsum >= nxx_len:
             break
-    return contig_len, total
+    greater_10k = [contig for contig in lens if contig > 1e4]
+    return contig_len, total, len(greater_10k)
+
+
+def html_from_judgement(good, bad):
+    '''return a formatted HTML string based on two judgment bool values
+    '''
```

Comment:
I love that this expresses a bit for both good and bad instead of just one. Would be good to add short descriptions of the args to the docstring